README.rst

```diff
@@ -114,7 +114,7 @@ Print the number of interceptions by the away team in a NCAA Football game
 
     from sportsreference.ncaaf.boxscore import Boxscore
 
-    championship_game = Boxscore('2018-01-08-georgia')
+    championship_game = Boxscore('2020-01-01-houston')
```

Comment:
Did you try out this code? I'm not sure if this game actually exists. We want to make sure the code on the README is all valid as most users will not give a project a chance if something is incorrect on the README.